deploy/cloudformation/manifest-pipeline-pipeline.yml

```diff
@@ -192,28 +202,27 @@ Resources:
       Path: /
   CodePipelineRolePolicy:
     Type: 'AWS::IAM::Policy'
-    DependsOn: CodePipelineTrustRole
-    Description: Setting IAM policy for the service role for AWS CodePipeline
     Properties:
       PolicyName: CodePipelineRolePolicy
       PolicyDocument:
         Statement:
-          - Action:
-              - 'ssm:Get*'
-              - 'ssm:Delete*'
-              - 'ssm:Put*'
-            Resource: !Sub "arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter/all/stacks/mellon*"
-            Effect: Allow
           - Action:
               - 'codebuild:StartBuild'
               - 'codebuild:BatchGetBuilds'
-            Resource: '*'
+            Resource: !Sub 'arn:aws:codebuild:${AWS::Region}:${AWS::AccountId}:project/${AWS::StackName}*'
+            Effect: Allow
+          - Action:
+              - 'cloudformation:DescribeStacks'
+              - 'cloudformation:DescribeChangeSet'
+              - 'cloudformation:CreateChangeSet'
```

Comment:
Is this a duplicate to what is in L187, and if so, which can be removed?

Reply:
My guess is 187 can be removed. Will check.

Reply:
I can confirm that both of these are required. Why there is a requirement for CreateChangeSet with the Serverless resource in the CloudFormationTrustRole isn't very clear. I suspect it has to do with the inner workings of the way it actually performs the transformation.